my-portfolio/resources/js/utils.js

```diff
@@ -0,0 +1,61 @@
+const data = [
+
+  {
+    name: 'project paper, rock, scissors',
+    description: 'A very popular game placed all across the world',
+    image: '/my-portfolio/resources/assets/projects/Paper-Rock-Scissors.JPG',
+    link: 'https://github.com/Phildeoner/Team-Project-PRS',
+  },
+
+  {
+    name: 'project calculator',
+    description: 'A replica of the famous iphone calculator',
+    image: '/my-portfolio/resources/assets/projects/Calculator.JPG',
+    link: 'https://github.com/akuutsang/Calculator',
+  },
+
+  {
+    name: 'project Etch-a-sketch',
+    description: 'A sketch pad',
+    image: '/my-portfolio/resources/assets/projects/Etch-A-Sketch.JPG',
+    link: 'https://github.com/akuutsang/Etch-A-Sketch',
+  },
+
+  {
+    name: 'Capstone project',
+    description: 'A collaborative work',
+    image: '/my-portfolio/resources/assets/projects/capstone.JPG',
+    link: 'https://phildeoner.github.io/Capstone-Project/index.html',
+  },
+
+  {
+    name: ' Landing page',
+    description: 'A simple landing page',
+    image: '/my-portfolio/resources/assets/projects/Landing-Page.jpg',
+    link: 'https://github.com/akuutsang/landingPage-portfolio',
+  },
+
+];
+
+function card() {
```

Comment:
This file is very neat. I'll suggest renaming this function since it is creating all the cards